crates/anvil/src/config.rs

```diff
@@ -525,15 +525,15 @@ impl NodeConfig {
     /// Returns the hardfork to use
     pub fn get_hardfork(&self) -> ChainHardfork {
         if self.odyssey {
-            return ChainHardfork::Ethereum(EthereumHardfork::Prague);
+            return ChainHardfork::Ethereum(EthereumHardfork::default());
```

Comment:
I think this is a fair assumption that Odyssey will continue to at least be the latest hardfork